Tests/MarkdownTests/Visitors/MarkupFormatterTests.swift

```diff
@@ -1626,4 +1626,31 @@ class MarkupFormatterMixedContentTests: XCTestCase {
         ]
         zip(expected, printed).forEach { XCTAssertEqual($0, $1) }
     }
+
+    func testDoxygenCommandsWithPrecedingNewlines() {
+        let expected = #"""
+            Does something.
+
+            \abstract abstract
+
+            \param x first param
+
+            \returns result
+
+            \note note
+
+            \discussion discussion
+            """#
+
+        let printed = Document(
+            Paragraph(Text("Does something.")),
+            DoxygenAbstract(children: Paragraph(Text("abstract"))),
+            DoxygenParameter(name: "x", children: Paragraph(Text("first param"))),
```

Comment:
How would this behave with two different parameters? _I_ wouldn't expect that to have blank lines in-between.

Reply:
Same reasoning as above.